src/sagas/background-refresh.js

```diff
@@ -0,0 +1,213 @@
+import {
+  all,
+  call,
+  put,
+  race,
+  select,
+  take,
+  takeEvery,
+  takeLatest,
+  throttle,
+} from 'redux-saga/effects'
+
+import * as Actions from '_/actions'
+import * as C from '_/constants'
+
+import AppConfiguration from '_/config'
+import { isNumber } from '_/utils'
+import { delay } from './utils'
+
+import {
+  fetchPoolsByCount,
+  fetchSinglePool,
+  fetchSingleVm,
+  fetchVmsByCount,
+  selectVmDetail,
+} from './index'
+import { getConsoleOptions } from './console'
+import { fetchIsoFiles } from './disks'
```

Comment:
Shouldn't it be imported from ./storageDomains instead of ./disks ?

Reply:
Yep you're quite right.  Good catch!